spec/user_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe User do
+ it { should have_valid(:first_name).when('Bob', 'Amanda') }
```

Comment:
(In this specific case) listen to the hound! Inconsistent indentation is amateur-hour.